components/wallet/wallet.js

```diff
@@ -29,18 +30,12 @@ const logLabel = 'Wallet';
 // Wallet will store address, private key and balance
 let wallets = {};
 
-/*  Dummy constructor for zilliqajs */
-// @dev: Will be replaced once zilliqa-js exposes utils without constructors
-const zilliqa = new Zilliqa({
-  nodeUrl: 'http://localhost:8888',
-});
-
 const createNewWallet = () => {
-  const pk = zilliqa.util.generatePrivateKey();
-  const address = zilliqa.util.getAddressFromPrivateKey(pk);
+  const pk = zCrypto.generatePrivateKey();
+  const address = zCrypto.getAddressFromPrivateKey(pk);
```

Comment:
Currently, `@zilliqa-js/account` supports BIP-39. I suggest you use the `Wallet` class in conjunction with a passphrase to generate these accounts, as that makes it repeatable between runs of `kaya`.

Reply:
Logged this as an issue. Will work on it next time. https://github.com/Zilliqa/kaya/issues/64